docs/src/pages/quick-start.md

```diff
@@ -26,14 +23,74 @@ function App() {
   return (
     // Provide the client to your App
     <QueryClientProvider client={queryClient}>
-      <Todos />
```

Comment:
i think it's a bit weird that we provide an example that is not really runnable. Rendering a `QueryClientProvider` without `children` isn't even valid jsx ....